### PR TITLE
Fix Argo link

### PR DESCRIPTION
**Pull request by simster7**

### CODE REVIEW

File: src/components/projects.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const PROJECTS: {[name: string]: { name: string, title: string; keywords:
         name: 'Argo Workflows' ,
         title: 'Workflows & Pipelines',
         description: 'Container native workflow engine for Kubernetes supporting both DAG and step based workflows.',
-        link: '/projects/argo',
+        link: '/projects/argo-workflows',
```
Comment (alexec) [inline src/components/projects.ts:6]: I don't think this fixes the issue though?

Comment (alexec) [reply src/components/projects.ts:6]: https://github.com/argoproj/argo-site/search?q=argoproj%2Fargo

Comment (alexmt) [reply src/components/projects.ts:6]: Issues was fixed by this PR: https://github.com/argoproj/argo-site/pull/42

**Comment by simster7** [comment]
Closed

**Comment by agilgur5** [comment]
Replaced by #42

**Pull request closed without merging**